app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
```

Comment:
Ensure that the import statements are grouped and ordered correctly. According to the checklist, standard library imports should come first, followed by local application/library specific imports. In this case, `import datetime` should be separated by a blank line from the `from app.errors import ...` statement .